contracts/CO30_Matias_BlackSmokeRising.blud_cordelia/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,5 @@
+Recently a guerilla movement has been gaining strength in the region. They've set up several camps along the river bank in preparation for an expansion into other areas.
+
+In order to prevent what could eventually become a military coup on the local government, the CIA has hired Synixe to move along the river bank neutralizing any enemy camps and destroying weapon caches within them.
+
+We've also been provided with some aerial images of the enemy camps.
```

Comment:
Would be better if it was a bit more complete and fleshed out.

Reply:
I think it's fine personally